chart/bin/setup.rb

```diff
@@ -0,0 +1,134 @@
+
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require_relative '../../config/environment'
+
+puts 'Adding and Installing Zookeeper and ZK'
+# Note, this needs to be wrapped in Bundler.with_clean_env otherwise the zookeeper install will fail
+Bundler.with_clean_env do
+  `bundle add zookeeper`
+  `bundle add zk`
+end
+
+require 'zk'
```

Comment:
My issue is here ... the first time this script runs, it fails

<img width="1485" alt="Screenshot 2020-03-23 at 15 52 03" src="https://user-images.githubusercontent.com/722117/77336835-01838900-6d20-11ea-8c0c-11f7aedbb56e.png">

I need to reload / refresh SOMETHING but I'm not sure what

Reply:
I imaging you need to reload the bundler env.  I've never added a gem to the bundle and then tried to immediatly use it.  I wonder what happens if you blow off the bundle and instead `gem install zookeeper zk`?

Reply:
that doesn't work, the zookeeper install fails ... I went round and round this many times!